crates/forge/tests/cli/test_cmd.rs

```diff
@@ -2390,3 +2391,260 @@ contract Dummy {
 
     assert!(dump_path.exists());
 });
+
+forgetest_init!(assume_no_revert_tests, |prj, cmd| {
```

Comment:
for new cheatcodes tests are added in `testdata/default/cheats` and picked up by `test_cheats_*` tests, please move them there
https://github.com/foundry-rs/foundry/blob/9fe891ab5babbdc2891c67d14d6c75ea1ca4b19c/crates/forge/tests/it/cheats.rs#L14-L16

see sample https://github.com/foundry-rs/foundry/blob/master/testdata/default/cheats/ExpectRevert.t.sol

Reply:
👍 yep, those can stay here